messaging-activemq/subsystem/pom.xml

```diff
@@ -98,6 +98,12 @@
         <dependency>
             <groupId>org.apache.activemq</groupId>
             <artifactId>artemis-server</artifactId>
+            <exclusions>
+                <exclusion>
+                    <groupId>org.apache.commons</groupId>
+                    <artifactId>commons-text</artifactId>
+                </exclusion>
+            </exclusions>
```

Comment:
I'm trying to understand why you are excluding it form artemis server as I can't find any direct dependecy on it. Maybe the right artefact to exclude is org.apache.commons:commons-configuration2

Reply:
Hm, it's not there in this version of Artemis. I didn't expect there to be difference from EAP 8. I removed this bit.

Reply:
:+1: